### ISSUE TITLE
Help Center: Modal help center has some squeezed images

### ISSUE THREAD

**Issue by zaguiini** [Automattic/wp-calypso#97141]
### Quick summary

In some articles, like the "Subscribe Block" one, we're seeing squeezed images:

![Image](https://github.com/user-attachments/assets/1300d29a-e142-4858-82eb-7563083b4384)

This happens because we're adding an explicit height to the image element. Removing the property fixes the aspect ratio and the image becomes legible again.

### Steps to reproduce

1. Open the Modal Help Center anywhere within Calypso;
2. Search for the "Subscribe Block" article;
3. Check the images are squeezed.

### What you expected to happen

The aspect ratio of the images should be respected.

### What actually happened

The image gets squeezed because of the wrong aspect ratio.

### Impact

Most (> 50%)

### Available workarounds?

Yes, easy to implement

### If the above answer is "Yes...", outline the workaround.

It's possible to update each article and and remove the explicit image height.

Another possible solution would be to add some CSS override that sets the `height` attribute to `auto`, but that feels hacky.

**Comment by jordesign** [comment]
Confirmed this issue is still occurring. It's a pretty rough look for the support experience - so placing this as high priority.

**Comment by jordesign** [comment]
@donalirl @happychait is this something we can keep in mind with producing new docs?

**Comment by donalirl** [comment]
Yes we already do but the issue was so widespread that I requested a fix here: https://github.com/Automattic/dotcom-forge/issues/10123 

Thank you for proactively fixing it!!

More info in p2rHU0-2aY-p2
